templates/controllers/modals/editorDecision/form/promoteForm.tpl

```diff
@@ -78,6 +78,11 @@
 				{load_url_in_div id="reviewAttachmentsGridContainer" url=$reviewAttachmentsGridUrl}
 			</div>
 		{/if}
+
+		<div id="libraryFileAttachments">
```

Comment:
It might be worth making this div collapsible, as I suspect this will take up a lot of real estate relative to its frequency of use.

Reply:
Done